npm-example/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "github-packages-test",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/atlassian/github-packages-test.git"
```

Comment:
According to [docs](https://help.github.com/en/packages/using-github-packages-with-your-projects-ecosystem/configuring-npm-for-use-with-github-packages#publishing-a-package-using-publishconfig-in-the-packagejson-file) probably should be only git. Not sure if git+https will work